Chaincase/Global.cs

```diff
@@ -388,6 +392,28 @@ private async Task AddKnownBitcoinFullNodeAsHiddenServiceAsync(AddressManager ad
 
         }
 
+
+        private IEnumerable<SmartCoin> QueuedCoins;
```

Comment:
The name of this field should be changed IMO (maybe `CoinsToBeQueued`), otherwise this will be hard to understand:
```
await Wallet.ChaumianClient.QueueCoinsToMixAsync(QueuedCoins);
QueuedCoins = null;
```

Reply:
is `SleepingCoins` still confusing?

Reply:
No it is not.